dom.bs

```diff
@@ -1488,7 +1488,7 @@ can be used to explore this matter in more detail.
 
 <h3 id=node-trees>Node tree</h3>
 
-<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}},
+<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}}, {{CDATASection}},
```

Comment:
The reason CDATASection is not mentioned here is because ShadowRoot isn't either. We cannot have it both ways.